src/main/scala/com/cognite/sdk/scala/v1/dataModelProperties.scala

```diff
@@ -85,11 +96,12 @@ object PropertyType {
     case object Boolean
         extends ArrayPropertyType[scala.Boolean, PropertyType.Boolean.type](PropertyType.Boolean)
     case object Int extends ArrayPropertyType[scala.Int, PropertyType.Int.type](PropertyType.Int)
-    case object Int32 extends ArrayPropertyType[scala.Int, PropertyType.Int.type](PropertyType.Int)
+    case object Int32
+        extends ArrayPropertyType[scala.Int, PropertyType.Int32.type](PropertyType.Int32)
     case object Int64
-        extends ArrayPropertyType[scala.Long, PropertyType.Bigint.type](PropertyType.Bigint)
+        extends ArrayPropertyType[scala.Long, PropertyType.Int64.type](PropertyType.Int64)
     case object Bigint
-        extends ArrayPropertyType[scala.Long, PropertyType.Bigint.type](PropertyType.Bigint)
+        extends ArrayPropertyType[scala.BigInt, PropertyType.Bigint.type](PropertyType.Bigint)
```

Comment:
Hi Jaime! Can you revert this back to scala.Long, it is what bigint stands for in dms API. Int64 and bigint stand for Long type afaik